packages/components/src/Modal/GenericModal/GenericModal.tsx

```diff
@@ -136,7 +136,7 @@ export const GenericModal = ({
       afterEnter={onAfterEnterHandler}
       afterLeave={onAfterLeaveHandler}
       data-generic-modal-transition-wrapper
-      onClick={(e: React.MouseEvent): void => e.stopPropagation()}
+      // onClick={(e: React.MouseEvent): void => e.stopPropagation()}
```

Comment:
this is no longer a props in newer version of headless-ui/react. I think worth confirming whether it is still needed